pkg/store/const.go

```diff
@@ -10,5 +10,5 @@ const (
 	// AliasesDir is the directory where tfversion stores the aliases.
 	AliasesDir = "aliases"
 	// TerraformBinaryName is the name of the Terraform binary.
-	TerraformBinaryName = "terraform"
```

Comment:
This is considered a breaking change, do we really want to do this? I doubt we have any external users relying on our packages, and perhaps we should've made this `internal/` to begin with. But just a heads-up.

Reply:
I hadn't considered there might be other users relying on this.
My reason for not exporting this const is that I'd prefer there to be one way to access the binary name, and since that depends on the client's OS I made this internal and created a func for it instead.
Making it private, and having a subsequent breaking changes for consumers of this package makes most sense to me.
Let me know how you'd like to proceed on this.

Reply:
Since we're pre-v1 we could version bump with a minor and accept the breaking change.